frontend/src/scenes/data-management/revenue/CurrencyDropdown.tsx

```diff
@@ -1,38 +1,65 @@
+    value: CurrencyCode | null
+    onChange: (currency: CurrencyCode | null) => void
+    visible?: boolean // Useful for stories to display the dropdown content
+}
+
+// These are the currencies that are most important to show first because they're used by the most customers.
+// Check our web analytics dashboard for the most popular countries from our visitors.
+const IMPORTANT_CURRENCIES: CurrencyCode[] = [
+    CurrencyCode.USD,
+    CurrencyCode.EUR,
+    CurrencyCode.GBP,
+    CurrencyCode.CAD,
+    CurrencyCode.INR,
+    CurrencyCode.CNY,
+    CurrencyCode.BRL,
+]
 
-export const CurrencyDropdown: React.FC<CurrencyDropdownProps> = (props) => {
+// All the other currencies, sorted by their "long name" in alphabetical order.
+const OTHER_CURRENCIES: CurrencyCode[] = (
+    Object.keys(CurrencyCode).filter(
+        (currency) => !IMPORTANT_CURRENCIES.includes(currency as CurrencyCode)
+    ) as CurrencyCode[]
+).sort((a, b) => {
+    return CURRENCY_SYMBOL_TO_NAME_MAP[a].localeCompare(CURRENCY_SYMBOL_TO_NAME_MAP[b])
+})
+
+const optionFromCurrency = (currency: CurrencyCode): LemonSelectOption<CurrencyCode> => {
+    return {
+        value: currency,
+        disabledReason: DISABLED_CURRENCIES[currency],
+        label: (
+            <span>
+                {CURRENCY_SYMBOL_TO_EMOJI_MAP[currency]} {CURRENCY_SYMBOL_TO_NAME_MAP[currency]} ({currency})
+            </span>
+        ),
+    }
+}
+
+// Computing these before hand is more efficient than computing them on the fly.
+const OPTIONS_FOR_IMPORTANT_CURRENCIES: LemonSelectOption<CurrencyCode>[] = IMPORTANT_CURRENCIES.map(optionFromCurrency)
+const OPTIONS_FOR_OTHER_CURRENCIES: LemonSelectOption<CurrencyCode>[] = OTHER_CURRENCIES.map(optionFromCurrency)
+
+export const CurrencyDropdown = ({ value, onChange, visible }: CurrencyDropdownProps): JSX.Element => {
     return (
         <LemonSelect
-            {...props}
-            options={
-                Object.keys(CurrencyCode).map((currency) => {
-                    const mappedCurrency = currency as CurrencyCode // Make TS happy, can't type on the function signature
-
-                    return {
-                        value: mappedCurrency,
-                        disabledReason: DISABLED_CURRENCIES[mappedCurrency],
-                        label: (
-                            <span>
-                                {CURRENCY_SYMBOL_TO_EMOJI_MAP[mappedCurrency]}{' '}
-                                {CURRENCY_SYMBOL_TO_NAME_MAP[mappedCurrency]}
-                            </span>
-                        ),
-                    }
-                }) as SelectOption[]
-            }
+            visible={visible}
+            value={value}
+            onChange={(newValue) => onChange(newValue as CurrencyCode | null)}
+            options={[
+                { options: OPTIONS_FOR_IMPORTANT_CURRENCIES, title: 'Most Popular' },
+                { options: OPTIONS_FOR_OTHER_CURRENCIES, title: 'Other currencies' },
```

Comment:
I didn't know we could do that, will surely use it on Page Reports, neat!

Reply:
Yeah, this is one of the coolest features we have on our `Select`, it's so easy to use. You can add a `footer` as well, I'm using it on the domain dropdown (main Web Analytics filter, if you wanna check)